task/sast-shell-check-oci-ta/0.2/sast-shell-check-oci-ta.yaml

```diff
@@ -0,0 +1,286 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: sast-shell-check-oci-ta
+  annotations:
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/tags: konflux
+  labels:
+    app.kubernetes.io/version: "0.2"
+spec:
+  description: |-
+    The sast-shell-check task uses [shellcheck](https://www.shellcheck.net/) to perform Static Application Security Testing (SAST). This task leverages a shellcheck wrapper ([csmock-plugin-shellcheck-core](https://github.com/csutils/csmock)) to run shellcheck on a directory tree.
+    ShellCheck is a static analysis tool, gives warnings and suggestions for bash/sh shell scripts.
+  params:
+    - name: CACHI2_ARTIFACT
+      description: The Trusted Artifact URI pointing to the artifact with
+        the prefetched dependencies.
+      type: string
+      default: ""
+    - name: IMP_FINDINGS_ONLY
+      description: Whether to include important findings only
+      type: string
+      default: "true"
+    - name: KFP_GIT_URL
+      description: Known False Positives (KFP) git URL (optionally taking
+        a revision delimited by \#). Defaults to "SITE_DEFAULT", which means
+        the default value "https://gitlab.cee.redhat.com/osh/known-false-positives.git"
+        for internal Konflux instance and empty string for external Konflux
+        instance. If set to an empty string, the KFP filtering is disabled.
+      type: string
+      default: SITE_DEFAULT
+    - name: PROJECT_NAME
+      description: Name of the scanned project, used to find path exclusions.
+        By default, the Konflux component name will be used.
+      type: string
+      default: ""
+    - name: RECORD_EXCLUDED
+      description: |
+        Whether to record the excluded findings (default to false).
+        If `true`, the excluded findings will be stored in `excluded-findings.json`.
+      type: string
+      default: "false"
+    - name: SOURCE_ARTIFACT
+      description: The Trusted Artifact URI pointing to the artifact with
+        the application source code.
+      type: string
+    - name: TARGET_DIR
+      description: Target directory in component's source code
+      type: string
```

Comment:
Will a single directory be sufficient for all projects?  I think it would be more flexible to accept a list of directories.  `run-shellcheck.sh` accepts multiple arguments already.